BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/GiniBankNetworkingScreenApiCoordinator.swift

```diff
@@ -515,7 +515,7 @@ extension GiniBankNetworkingScreenApiCoordinator: SkontoCoordinatorDelegate {
     }
 
     func didTapDocumentPreview(_ coordinator: Coordinator, _ skontoViewModel: SkontoViewModel) {
-        let viewController = DocumentPagesViewController()
+        let viewController = DocumentPagesViewController(screenTitle: DocumentPagesViewModel.screenTitle)
```

Comment:
[clarification] why do we need `screenTitle ` for Skonto flow?

Reply:
In Skonto flow we have constant title from localized strings, and in TD flow we will inject fileName there
It injects here, because screen is showed earlier, than setData with viewModel called (after API request)